godotproject/DeathPlane.gd

```diff
@@ -1,6 +1,17 @@
 extends Area2D
 
+const level0_start_postion :=  Vector2(121, 896)
+const level1_start_postion := Vector2(121, 256)
```

Comment:
Can we change this to be:
```suggestion
export (Vector2) var start_position
```
and then set it inside each Level scene?

Reply:
sounds good